config/service-brokers/compose/catalog.json

```diff
@@ -30,7 +30,38 @@
         "databaseType": "mongodb"
       }
     }]
-  },{
+  }, {
+    "id": "1356eeeb-7c5d-4d9d-9a04-c035a2c709b3",
+    "name": "redis",
+    "bindable": true,
+    "description": "Redis instance",
+    "requires": [],
+    "tags": [
+      "redis",
+      "compose"
+    ],
+    "metadata": {
+      "displayName": "Redis",
+      "imageUrl": "https://redis.io/images/redis-logo.svg",
+      "longDescription": "Redis instance",
+      "providerDisplayName": "GOV.UK PaaS",
+      "documentationUrl": "https://compose.com/databases/redis",
+      "supportUrl": "https://www.cloud.service.gov.uk/support.html"
+    },
+    "plans": [{
+      "id": "a8574a4b-9c6c-40ea-a0df-e9b7507948c8",
+      "name": "tiny",
+      "description": "256MB Storage / 256MB RAM",
+      "compose": {
+        "units": 1,
+        "databaseType": "redis"
+      },
+      "metadata": {
+        "displayName": "Redis Tiny",
```

Comment:
Should this say anything about it being "storage mode"? If we plan to introduce a "cache mode" plan afterwards and it's not advisable to use this plan for caching. Or are any of the words to describe "non-cache" more confusing than they are helpful? /cc @bleach @alext for thoughts

Reply:
heh we had "Redis (storage mode)" in the displayname/description originally but thought it might be confusing without the complementary "Redis (cache mode)" descriptions since this is really a compose thing not a redis thing.

Maybe we could add a note to the story that enables provisioning cache-mode instances to ensure the descriptions are clearly labeled?